src/Card/README.md

```diff
@@ -670,3 +671,53 @@ it is meant to be used as a single horizontal row of Cards, not as a grid. See C
   </Card>
 </CardDeck>
 ```
+
+### Card status
+
+Note that in the example below, the content of `Card` is wrapped inside `Card.Body`. The `d-flex` class is added for the `horizontal` orientation to achieve horizontal variant. The `flex-column` class is added for the main `Card` component.
+
+```jsx live
+() => {
+  const [orientation, setOrientation] = useState('vertical');
+  
+  const handleChangeOrientation = (e) => setOrientation(e.target.value);
+  
+  return (
+    <>
+      <Form.Group>
+        <Form.Label>Choose orientation</Form.Label>
+        <Form.RadioSet
+          name="orientation"
+          onChange={handleChangeOrientation}
+          value={orientation}
+        >
+          <Form.Radio value="vertical">Vertical</Form.Radio>
+          <Form.Radio value="horizontal">Horizontal</Form.Radio>
+        </Form.RadioSet>
+      </Form.Group>
+      <Card orientation={orientation} className="flex-column">
+        <Card.Body className={orientation === 'horizontal' ? 'd-flex' : ''}>
+          <Card.Header
+            title="Card title"
+          />
+          <Card.Section
+            title="Section title"
+          >
+            This is a wider card with supporting text below as a natural lead-in to
+            additional content. This card has even longer content than the first to
+            show that equal height action.
+          </Card.Section>
+          <Card.Footer className="justify-content-end">
```

Comment:
I believe the buttons in the `Card.Footer` are already justified for `flex-end` in the vertical orientation; should we only apply this class name for the horizontal orientation? That said, this also raises the (design) question of whether the content inside of `Card.Footer` should have `justify-content: flex-end` for the horizontal orientation already 👀

Reply:
Until we have an answer I added `className={!isVertical ? 'justify-content-end' : ''}`